Add NOTES.md (#39)
openvinotoolkit/openvino.genai: llm_bench/python/README.md

```bash
python ./benchmark.py -m models/llama-2-7b-chat/pytorch -d CPU --torch_compile_backend openvino
```

# Notes

> > If you encounter any issues, you can check out the **(./NOTES.md)**.

Reviewer: ```suggestion
If you encounter any error messages, you can check the **[NOTES.md](./NOTES.md)**  which includes solutions for some errors.
```

> If you encounter any error messages, you can check the **[NOTES.md](./NOTES.md)**  which includes solutions for some errors.

Reviewer: ```suggestion
If you encounter any error messages, please check **[NOTES.md](./NOTES.md)**  which provides solutions to some known errors.
```
